src/sickle.ts

```diff
@@ -290,6 +280,40 @@ class Annotator {
     this.indent--;
   }
 
+  private expandSymbolsFromExportStar(exportDecl: ts.ExportDeclaration): string[] {
+    let typeChecker = this.program.getTypeChecker();
+
+    // Gather the names of local exports, to avoid reexporting any
+    // names that are already locally exported.
+    let locals =
+        typeChecker.getSymbolsInScope(this.file, ts.SymbolFlags.Export | ts.SymbolFlags.Alias);
+    let localSet: {[name: string]: boolean} = {};
+    for (let local of locals) {
+      localSet[local.name] = true;
+    }
```

Comment:
This guy doesn't depend on exportDecl, I wonder if there is a better way to put it, instead of rebuilding it per decl.